sql/jjitTablesDeclare.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE jjit_job_offers (
+	id INT IDENTITY(1,1) PRIMARY KEY,
+	title NVARCHAR(100),
+	company_name NVARCHAR(100),
+	company_size INT,
+	company_link NVARCHAR(200),
+	job_level INT,
+	company_location NVARCHAR(100),
+	link NVARCHAR(200),
+	salary_uop_min DECIMAL(18, 2),
+	salary_uop_max DECIMAL(18, 2),
+	salary_b2b_min DECIMAL(18, 2),
+	salary_b2b_max DECIMAL(18, 2),
```

Comment:
Idea: Converting salary values to BIGINTS and storing adequate amount of decimal places in the currency table.